src/main/kotlin/org/elm/ide/inspections/MissingCaseBranchAdder.kt

```diff
@@ -65,16 +61,25 @@ class MissingCaseBranchAdder(val element: ElmCaseOfExpr) {
         var start = elements.first().prevSibling.prevSibling
 
         // Add the virtual open token if there isn't one already
-        if (existingBranches.isNotEmpty()) {
+        if (existingBranches.isNotEmpty() && element.text[element.textLength - 1] != '\n') {
```

Comment:
```suggestion
        if (existingBranches.isNotEmpty() && element.text.endsWith('\n')) {
```
Would this work the same way?

or 
```
        if (existingBranches.isNotEmpty() && element.text.last() != '\n') {
```

Reply:
Ah, thanks for pointing that out. I'm new to Kotlin, so I missed those parts of the API :slightly_smiling_face: